client/clientimpl_test.go

```diff
@@ -1527,13 +1537,35 @@ func TestUpdatePackages(t *testing.T) {
 	errorOnCallback.errorOnCallback = true
 	tests = append(tests, errorOnCallback)
 
+	// Run each test case.
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			verifyUpdatePackages(t, test)
 		})
 	}
 }
 
+// Mock download server that checks headers in the request.
+func createDownloadServWithAuth(t *testing.T) *httptest.Server {
```

Comment:
What's the purpose of this func? I don't see it used anywhere?

Reply:
Removed unused `function createDownloadServWithAuth` as it wasn’t utilized in tests